static/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -94,6 +94,7 @@ function EventOrGroupExtraDetails({
               pathname: issuesPath,
               query: {
                 query: `logger:${logger}`,
+                project: [project.id],
```

Comment:
this should be using a https://github.com/getsentry/sentry/blob/master/static/app/components/globalSelectionLink.tsx which will add the project to the query for you